students/N0vA/lesson_02/series.py

```diff
@@ -0,0 +1,76 @@
+#### Series Functions ####
+
+def fibonacci(n):
+    """Function computes the nth value in the Fibonacci 
+    series"""
+
+    if n <= 1:
+        return n
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+
+
+def lucas(n):
+    """Function that computes the nth value in the Lucas Number
+    series"""
+
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n-1) + lucas(n-2)
+
+
+def sum_series(n, n_0 = 0, n_1 = 1):
```

Comment:
You don't want white spaces around the equal sign assignment operators in this case.